po/es.po

```diff
@@ -135,19 +134,19 @@ msgstr "`break` y `continue`"
 
 #: src/SUMMARY.md src/control-flow-basics/blocks-and-scopes.md
 msgid "Blocks and Scopes"
-msgstr ""
+msgstr "Funciones"
```

Comment:
Wrong translation for "Blocks and scopes". I guess the contents of this is mostly functions, but the purpose is to teach how scopes work in Rust.

Reply:
Yeah, is wrong